spec/openapi3.yaml

```diff
@@ -2805,6 +2835,55 @@ components:
       allOf:
         - $ref: "#/components/schemas/EmbeddedTransactionDTO"
         - $ref: "#/components/schemas/TransferTransactionBodyDTO"
+
+    # Schemas - Metadata
+    KeyMetadataDTO:
+      type: object
+      required:
+      - key
+      - metadata
+      properties:
+        key:
```

Comment:
To be consistent with other features, it might make sense to return it in an array format. You can use $ref: "#/components/schemas/UInt64" type.